server/web/errors/index.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+exports.register = function(server, options, next) {
+
+  server.ext("onPreResponse", function (request, reply) {
+
+    var response = request.response;
+    if (!response.isBoom) {
+        return reply.continue();
+    }
+
+    // Replace error with friendly HTML
+    var error = response;
```

Comment:
nitpick but looks like 

1. `request.response` becomes `response` 
2. `response` is used once 
3. `response` becomes `error`
4. `error` is used once

I'm all for shortening the call paths just seems like could do it with a few less vars. Would enjoy hearing how you see it.

```javascript
if (!request.response.isBoom) {
  return reply.continue();
}

var statusCode = request.response.output.statusCode;
```